### ISSUE TITLE
Menus are not centered

### ISSUE THREAD

**Issue by Alexander-Wilms** [vcmi/vcmi#3015]
**Describe the bug**
Menus/windows have different margins on the left and right. In OH3, these are centered.

**Game logs**
na

**To Reproduce**
1. Start scenario in OH3 and create a savegame
2. Start scenario in VCMI and create a savegame

**Expected behavior**
Both times the windows are centered.

**Actual behavior**
In VCMI's case, they are not centered.

**Did it work earlier?**
Unknown

**Screenshots**
OH3:
![image](https://github.com/vcmi/vcmi/assets/3226457/54fa6119-cdd4-4ded-b981-de2fa0096de6)

![image](https://github.com/vcmi/vcmi/assets/3226457/aada28cd-b1a1-4624-91be-b74f5a84ca69)
VCMI:
![image](https://github.com/vcmi/vcmi/assets/3226457/b0a0a28d-342a-4dd2-875d-d99b9d24b7e0)

![image](https://github.com/vcmi/vcmi/assets/3226457/3fa74f57-dd40-4800-9a63-cdee58889e39)

![image](https://github.com/vcmi/vcmi/assets/3226457/06cbbb11-c9fa-4a38-b8e8-58a016958080)

**Version**
 - OS: openSUSE Tumbleweed
 - Version: develop

**Additional context**

**Comment by dydzio0614** [comment]
i will mark it as "enhancement" since this is HD mod, not H3 feature and we allow ourselves to implement extensions via own design
looks better in HD mod version anyway

**Comment by Alexander-Wilms** [comment]
Indeed, the screenshots show the OH3 demo. The H3 Complete screens are not centered either.

Some of the involved bitmaps:

`SCSelBck.bmp`: 
![image](https://github.com/vcmi/vcmi/assets/3226457/4f9882be-217a-4a42-ad13-de7add77285b)

`camcust.bmp`:
![image](https://github.com/vcmi/vcmi/assets/3226457/45cb173c-e284-4ec1-b06d-2c74b4f7c529)
